packages/coding-standards/src/Sniffs/ConstantVisibilityRequiredSniff.php

```diff
@@ -4,10 +4,10 @@
 
 namespace Shopsys\CodingStandards\Sniffs;
 
+use function array_reverse;
```

Comment:
na tohle mam takovej ugly hack, aby to bylo "kontextove u sebe"

viz. https://gitlab.com/uxf/fan/-/blob/main/ecs.php#L28

Reply:
I thought we have the rule for grouping types of `use` applied...

Reply:
I don't think there was any rule handling this. This is some internal change in ECS or CS Fixer new version... It's not the first time I see this change after upgrading ECS.

I'll try @janatjak suggestion and check the results. This sorting triggers me, but I am not in position to decide if it's ok or not.

Reply:
I prefer to have imports sorted and grouped to `class`, `function`, `constant` parts (I believe it should not generate many changes in that case)

Reply:
@janatjak We had the rule set like this:
```php
    $services->set(OrderedImportsFixer::class)
        ->call('configure', [['sort_algorithm' => 'alpha', 'imports_order' => ['class', 'function', 'const']]]);
```
But it was overrident by some set. Imports was a bit messy :D